space-plugins/nextjs-starter/src/pages/index.tsx

```diff
@@ -14,15 +12,8 @@ type UserInfo = {
 	user: User;
 };
 
-type HomeProps = {
-	userInfo: UserInfo;
-	spaceId: string;
-	userId: string;
-	isAdmin: boolean;
-};
-
-export default function Home(props: HomeProps) {
-	const toolContext = useToolContext();
+export default function Home() {
+	const { completed } = useAppBridge({ type: 'space-plugin', oauth: true });
```

Comment:
I am wondering if it is possible to create config file for this too? where the use can enable and disable app bridge

Reply:
I didn't consider the configurability in this implementation, as there's still the legacy nextjs project that doesn't use App Bridge. In case of Nuxt starter, we were already using nuxt-base in other projects, so we had to introduce the config to keep the backward compatibility. Now that App Bridge will be kind of the new default and it won't be visible and very quick to run, I think we can keep it as-is.